map.css

```diff
@@ -6,7 +6,7 @@ body {
 }
 
 body.events {
-  background-color: #ed135c;
+  /* background-color: #ed135c; */
```

Comment:
This file is used by both index.html and events.html, and the change should only be made for events.html.

Reply:
In index.html, the body tag does not have the 'events' class. So, this change would apply to events.html, family_separation_events.html, marchonpolls_events.html, but not to index.html. What do you think, should we keep it?